packages/expect/src/jest-extend.ts

```diff
@@ -85,15 +87,21 @@ function JestExtendPlugin(
             const thenable = result as PromiseLike<SyncExpectationResult>
             return thenable.then(({ pass, message, actual, expected }) => {
               if ((pass && isNot) || (!pass && !isNot)) {
-                throw new JestExtendError(message(), actual, expected)
+                const errorMessage = customMessage
+                  ? `${customMessage}: ${message()}`
```

Comment:
default matchers ignore the matcher message in this case, so I think we should also ignore it here


```suggestion
                  ? customMessage
```